config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-presubmits-main.yaml

```diff
@@ -17,11 +17,11 @@ presubmits:
         - "./scripts/ci-test.sh"
         resources:
           limits:
-            cpu: 4
-            memory: 4Gi
+            cpu: 16
```

Comment:
do we have machines with 16 CPUs? @upodroid @ameukam

Reply:
This will fail. Unfortunately, there is no proper doc to describe the capacity available.

Reply:
7 will be the maximum on GCP.

Reply:
Fixup: https://github.com/kubernetes/test-infra/pull/35947